src/presentations/Cart.js

```diff
@@ -0,0 +1,137 @@
+                                <span className="process-btn plus" aria-label={'increased item quantity to'+item.qty} onClick={(event)=>this.increaseQty(event,item,i)}>
+                                </span>
+                                <span className="price-area">X <p aria-label={'Rs.'+item.price} title={item.price}>{item.price}</p></span>
+                            </div>
+                        </div>
+                        <div className="cart-item-totalprice">
+                            <p aria-label={item.finalPrice}>{item.finalPrice = item.qty * item.price}</p>
+                        </div>
+                        </li>))}
+                        </ul>
+                        </div>
+                        <div className="site-promoto">
+                            <div className="promo-img">
+                                <LazyLoad height={25} once>
+                                    <img src={window.location.origin + '/static/images/lowest-price.png'} alt="Lowest Price Guaranteed"></img>
+                                </LazyLoad>
+                            </div>
+                            <div className="promo-text">
+                                <p>You won't find it cheaper anywhere</p>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="footer-area">
+                    <div><p>Promo code can be applied on payment page</p></div>
+                    <div aria-label={` ${'Proceed to checkout with '+totalCartValue+' rupees.'} `} onClick={checkoutComplete} className="footer-btn modal-horizontal-space">
+                        <div className="checkouttxt">Proceed to checkout</div>
+                        <div className="price-info">Rs.{totalCartValue} 
+                            <span className="arrow right">
+                            </span>
+                        </div>
+                    </div>
+                    </div>
+                </div>) : (<div>
+                    <div className="modal-horizontal-space no-cart-items">
+                        <div>
+                        <h2>No items in your cart</h2><br /> <p>Your favorite items are just a click away.</p>
+                        </div>
+                    </div>
+                    <div className="modal-horizontal-space footer-shop-btn">
+                        
+                        <button aria-label="Start shopping" onClick={!hideHeader ? closeModal : checkoutComplete} className="btn full-width-btn">Start Shopping</button>
+                    </div>
+                    </div>)}
+            </section>
+        </main>)
+    }
+}
+function mapStateToProps(state){
+    return{
+        productInfo: state.productReducer
+    }
+}
+const mapDispatchToProps = (dispatch,ownProps) => {
+    return {
+        resetCartData: ()=>{dispatch(resetCartData())}
+    }
+}
+export default connect(mapStateToProps,mapDispatchToProps)(Cart);
```

Comment:
add line break
extract independent jsx and make it a separate component. Make your components smaller

Reply:
added line break